bookdown/_output.yml

```diff
@@ -29,3 +29,5 @@ bookdown::pdf_book:
   latex_engine: xelatex
   highlight: haddock
   keep_tex: yes
+  documentclass: srcbook
+  fontsize: 9pt
```

Comment:
Why are we doing this via yaml? Wouldn't it be better to put stuff like this in the preamble?

Reply:
Otherwise we have two `\documentclass()` class in the final .tex (pandoc default + custom setting).